packages/venia-ui/lib/components/CartPage/PriceAdjustments/GiftOptions/giftOptions.gql.js

```diff
@@ -11,11 +11,11 @@ import { gql } from '@apollo/client';
  */
 const GET_GIFT_OPTIONS = gql`
     query getGiftOptions($cartId: String!) {
-        cart(cart_id: $cartId) @client {
+        cart(cart_id: $cartId) {
             id
-            include_gift_receipt
-            include_printed_card
-            gift_message
+            include_gift_receipt @client
+            include_printed_card @client
+            local_gift_message @client
```

Comment:
Why the rename here?

Reply:
[`gift_message`](https://devdocs.magento.com/guides/v2.4/graphql/queries/cart.html#GiftMessage) is a field in `Cart` now, which didn't have the same type as our local defined one; validator threw an error. I don't think it did at the time of writing, so this ideally this should be re-factored to use the real fields, now that they're available. Could be a good first task for community, happy to write that up if you agree.

Reply:
Yea I think the original intent behind the message was to actually hook this up :) As is, it doesn't actually do anything which might be surprising.